services/tasks/project/api/routes.py

```diff
@@ -136,18 +149,55 @@ def populate_db():
                         db.session.add(
                             Task(
                                 created_at=created_at,
+                                updated_at=updated_at,
                                 creator=creator,
                                 labels=labels,
                                 language=language,
                                 num_of_comments=num_of_comments,
                                 num_of_reactions=num_of_reactions,
                                 title=title,
-                                url=url
+                                url=url,
+                                task_type=task_type
                             )
                         )
                         db.session.commit()
                     except exc.IntegrityError:
                         db.session.rollback()
                         return jsonify(response_object), 400
+    return jsonify(response_object), 201
 
-    return jsonify(response_object), 201
\ No newline at end of file
+@tasks_blueprint.route('/tasks/rice/<task_id>', methods=['GET'])
+def calculate_rice_score(task_id):
+    response_object = {
+        'status': 'fail',
+        'message': f'task_id {task_id} does not exist'
+    }
+    task = Task.query.filter_by(id=int(task_id)).first()
+    if not task:
+        return jsonify(response_object), 404
+    else:
+        priority = Priority()
+        elements = {
+            'reach': request.args.get('reach', type = float),
+            'impact': request.args.get('impact', type = float),
+            'confidence': request.args.get('confidence', type = float),
+            'effort': request.args.get('effort', type = float),
```

Comment:
What happens if I give you a string?

Reply:
I could either return an error or we can convert to a float.

Reply:
Added a test to verify this works as expected. You can provide a float or a string. If the latter, the string gets converted to a float.